packages/vdaf/src/index.ts

```diff
@@ -19,22 +19,22 @@ export interface Vdaf<
 
   measurementToInputShares(measurement: Measurement): Promise<Buffer[]>;
 
-  initialPrepareMessage(
+  initialPrepareState(
     verifyKey: Buffer,
     aggId: number,
     aggParam: AggregationParameter,
     nonce: Buffer,
     inputShare: Buffer
-  ): Promise<PrepareMessage>;
+  ): Promise<PrepareState>;
 
   prepareNext(
-    prepareMessage: PrepareMessage,
+    prepareState: PrepareState,
     inbound: Buffer | null
   ):
-    | { prepareMessage: PrepareMessage; prepareShare: Buffer }
+    | { prepareState: PrepareState; prepareShare: Buffer }
     | { outputShare: OutputShare };
 
-  prepSharesToPrepareMessage(
+  prepSharesToPrepareState(
```

Comment:
This should keep the name prepSharesToPrepareMessage. See https://cfrg.github.io/draft-irtf-cfrg-vdaf/draft-irtf-cfrg-vdaf.html#section-5.2-6.3